ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -141,9 +141,9 @@
       (let [updated-concept (update-collection-concept context concept bulk-update-params)
             warnings (validate-and-save-collection context updated-concept)]
         (data-bulk-update/update-bulk-update-task-collection-status context task-id
```

Comment:
Indentation nit:
`context` should be on the next line indented one space. If there end up being multiple lines of parameters they should line up at the same level as `context`.